.storybook/addons.js

```diff
@@ -1 +1,2 @@
 import '@storybook/addon-knobs/register';
+import '@storybook/addon-options/register';
```

Comment:
Please be careful with removing newline at the end of the file. Editors often have a setting to auto add a newline.
If you can add a newline, it would be awesome.